Formula/qt-unixodbc.rb

```diff
@@ -0,0 +1,75 @@
+class QtUnixodbc < Formula
+  desc "Qt SQL Database Driver"
+  homepage "https://www.qt.io/"
+  url "https://download.qt.io/official_releases/qt/6.0/6.0.1/submodules/qtbase-everywhere-src-6.0.1.tar.xz"
+  sha256 "8d2bc1829c1479e539f66c2f51a7e11c38a595c9e8b8e45a3b45f3cb41c6d6aa"
+  license all_of: ["GPL-2.0-only", "GPL-3.0-only", "LGPL-2.1-only", "LGPL-3.0-only"]
+
+  depends_on "cmake" => [:build, :test]
+
+  depends_on "qt"
+  depends_on "unixodbc"
+
+  conflicts_with "qt-libiodbc",
+    because: "qt-unixodbc and qt-libiodbc install the same binaries"
+
+  def install
+    cd "src/plugins/sqldrivers" do
+      system "qmake", "--", "ODBC_PREFIX=#{Formula["unixodbc"].opt_prefix}"
+      system "make", "sub-odbc"
+      (share/"qt").install "plugins/"
+    end
+    Pathname.glob(share/"qt/plugins/sqldrivers/*.dylib") do |plugin|
```

Comment:
```suggestion
    Pathname.glob(share/"qt/plugins/sqldrivers/#{shared_library("*")}") do |plugin|
```
This will make it portable to linuxbrew-core.